### PR TITLE
Wording checkout delivery

### PR DESCRIPTION
**Pull request by aleeks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Wording checkout delivery
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-3026
| How to test?  | active gift wrapping, define a price & see checkout page

delivery

### CODE REVIEW

File: classes/checkout/CheckoutDeliveryStep.php [hunk 1]
```diff
@@ -180,9 +180,9 @@ public function render(array $extraParams = array())
                     'allowed' => $this->isGiftAllowed(),
                     'isGift' => $this->getCheckoutSession()->getGift()['isGift'],
                     'label' => $this->getTranslator()->trans(
-                        'I would like my order to be gift wrapped'.$this->getGiftCostForLabel(),
-                        array(),
-                        'Checkout'
+                        'I would like my order to be gift wrapped %cost%',
```
Comment (xBorderie) [inline classes/checkout/CheckoutDeliveryStep.php]: @AlexEven Unsure about this...

Comment (aleeks) [inline classes/checkout/CheckoutDeliveryStep.php]: @xBorderie to me defense, http://forge.prestashop.com/browse/BOOM-3026 :smiley:

Comment (xBorderie) [inline classes/checkout/CheckoutDeliveryStep.php]: Oh, indeed, %cost% contains "(additional cost of %giftcost% %taxlabel%)" ([Crowdin link](https://crowdin.net/translate/prestashop-official/33242/en-fr#5719742)), not just the number :)
Ok then! Go ahead ;)

Comment (aleeks) [inline classes/checkout/CheckoutDeliveryStep.php]: Yes %cost% is generated by an other translation!

Comment (aleeks) [inline classes/checkout/CheckoutDeliveryStep.php]: SO LETSGOTOMERGEIT !

**Pull request merged**